apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -135,10 +173,17 @@ private void MaintainPool()
         /// Provides a snapshot of statistics for a database-specific pool.
         /// </summary>
         /// <returns>A snapshot of statistics for this pool.</returns>
-        public DatabaseStatistics GetStatisticsSnapshot(DatabaseName databaseName)
+        public DatabaseStatistics GetStatisticsSnapshot(DatabaseName databaseName) =>
+            GetStatisticsSnapshot(databaseName, null);
+
+        /// <summary>
+        /// Provides a snapshot of statistics for a database-databaseRole-specific pool.
+        /// </summary>
+        /// <returns>A snapshot of statistics for this pool.</returns>
+        public DatabaseStatistics GetStatisticsSnapshot(DatabaseName databaseName, string databaseRole)
```

Comment:
Let's make the new overload just accept the new key type, and create a new key in the method above.

Reply:
(Ditto for the other methods.)